docs/administration/authentication/active-directory-authentication/index.md

```diff
@@ -44,6 +44,18 @@ Octopus is built on top of HTTP.sys, the same kernel driver that IIS is built on
 When the link is clicked, it redirects to a page which is configured to tell HTTP.sys to issue the browser challenge. The browser and HTTP.sys negotiate the authentication just like an IIS website would. The user principal is then passed to Octopus. Octopus will then query Active Directory for other information about the user.
 :::
 
+### Using Kerberos {#ActiveDirectoryauthentication-UsingKerberos}
+
+By default `IntegratedWindowsAuthentication` will try to use Kerberos first, and then fall back to `NTLM` if it fails.
+
+Please note that this may require additional configuration to work correctly in your network. Generally, this will require a Service Principal Name (SPN) to be set for the `HTTP` service class, using the machine name. For example, if you have an Octopus Deploy hosted on the `od.mydomain.local` domain name, that is installed on a machine called `winserv2019` then the corresponding SPN will be needed:
```

Comment:
Should it say `HTTP`/`HTTPS` ?

Reply:
You'd think so! But no - this is a service class not a protocol. From the support article: 
![image](https://user-images.githubusercontent.com/579657/81123813-21d77200-8f73-11ea-8589-5fb289f48c39.png)